components/table/src/test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { createMatchers } from 'jest-emotion';
+import * as emotion from 'emotion';
+import Component from '.';
+
+expect.extend(createMatchers(emotion));
+
+describe('Table', () => {
+  const titles = ['Heading 1', 'Heading 2', 'Heading 3', 'Heading 4'];
+  const rows = [
+    ['Content 1-1', 'Content 1-2', 'Content 3', 'Content 4'],
```

Comment:
heh - arguably the last two items on this row are wrong :grin: as they should be `Content 1-3` and `Content 1-4`